apps/storefront/src/shared/service/b2b/graphql/register.ts

```diff
@@ -225,3 +410,13 @@ export const getBCStoreChannelId = () =>
     query: getStoreChannelId,
     variables: { storeHash, bcChannelId: channelId },
   });
+
+export const createBCCompanyUser = (data: Partial<CreateCustomer>) =>
```

Comment:
🤔 are all the properties of the `CreateCustomer` optional? I'd imagine some (`email`?) might be required.